src/pages/about-page/about-page.module.css

```diff
@@ -0,0 +1,104 @@
+.about {
+  padding: 40px 0 20px;
+}
+
+.about__inner {
+  display: flex;
+  justify-content: space-between;
+  align-items: start;
+  gap: 80px;
+}
+
+.about__inner-text {
+  display: flex;
+  flex-direction: column;
+  max-width: 45%;
+  padding-bottom: 70px;
+}
+
+.about__inner a {
+  color: #111;
+  font-weight: 400;
+  text-decoration: underline;
+}
+
+.about__inner p {
+  font-weight: 400;
+  font-size: 1.37rem;
+  line-height: 32px;
+  margin-bottom: 20px;
+}
+
+.about__inner p:last-child {
+  margin-bottom: 0;
+}
+
+.about__inner-image {
+  max-width: 47%;
+  width: 42.6%;
+  height: 489px;
+  margin-top: 73px;
+}
+
+.about__inner-image img {
+  width: 100%;
+  height: 100%;
+  border-radius: 40px;
+  object-fit: cover;
+}
+
+@media (max-width: 1120px) {
+
+  .about__inner {
+    gap: 40px;
```

Comment:
Please correct gaps according to design
![image](https://github.com/greyzerua/fe-react-2024/assets/98031768/dbf7f7e7-11f3-49df-9007-d860a86de536)

Reply:
Same for the desktop version 
![image](https://github.com/greyzerua/fe-react-2024/assets/98031768/a3b2c4cd-6ed4-4967-bca0-7a9b26086087)